corehq/apps/export/views/download.py

```diff
@@ -381,7 +381,7 @@ def poll_custom_export_download(request, domain):
                          'exception_name': e.exception_name})
 
             return JsonResponse({
-                'error': _("Download task failed to start."),
+                'error': _("Failed to download export."),
```

Comment:
Is there any direction HQ could give the user? Should they, say, try again in a few minutes?

Reply:
![Screen Shot 2022-02-01 at 8 20 20 AM](https://user-images.githubusercontent.com/15785053/152007366-ad773e78-ce31-4e40-81bb-47f00587a9ed.png)
It looks like the frontend or just somewhere further up the chain already adds that. Does that seem good enough?

Reply:
yep, that's good